src/libs/actions/Task.ts

```diff
@@ -798,6 +798,13 @@ function setAssigneeValue(
         if (!shareToReportID && !skipShareDestination) {
             setShareDestinationValue(report?.reportID ?? '-1');
         }
+    } else {
```

Comment:
Can we flip the logic here so that it's cleaner?

if (isCurrentUser) {
...
} else {
...
}

Otherwise, this looks good to me.

Reply:
@srikarparsi i updated